src/test/java/org/folio/rest/impl/TenantRefAPITest.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.rest.impl;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitParamsRunner.class)
+public class TenantRefAPITest {
+  @Parameters({
+    "null  , 1.22.3, true",
+    "1.3.2 , 1.22.3, true",
+    "1.22.2, 1.22.3, true",
+    "1.22.3, 1.22.3, false",
+    "1.22.4, 1.22.3, false",
+  })
+  @Test
+  public void isNew(String migratingFromVersion, String featureVersion, boolean expected) {
+    if ("null".equals(migratingFromVersion)) {
+      migratingFromVersion = null;
+    }
```

Comment:
You can annotate `String migrationFromVersion` with `@Nullable` annotation

Reply:
Thanks, fixed.